frameworks/digital-outcomes-and-specialists/questions/services/designer.yml

```diff
@@ -12,3 +12,5 @@ questions:
   - designerDayRate
 
 empty_message: You haven't added a designer
+
+hint: Set up a team for successful delivery. Remove obstacles, track progress, facilitate meetings and help the team organise itself.
```

Comment:
In the doc it is
> Design interactions that support user needs. This may involve sketching, design production, hand-coded HTML and CSS, and rapid prototyping.

…but this doesn’t cover service design and graphic design. We’re going to come up with an alternative.